src/pb_stub.h

```diff
@@ -74,6 +77,10 @@ class Logger {
   // Log a message.
   void Log(const std::string& msg) { std::cerr << msg << std::endl; }
```

Comment:
I think we probably need to redirect all the existing logging in Python backend to the server too right? Otherwise, the logging info in the stub process doesn't obey the server's logging settings. You should also make all the `log_<level>` function in the triton_python_backend_utils part of this class.

Reply:
Migrated existing LOG_INFO macro to send messages through the new infrastructure. I also created LOG_WARN, LOG_ERROR, and LOG_VERBOSE macros so we can send future logs with setting-level precision.

Reply:
Thank you!